package/MDAnalysis/core/groups.py

```diff
@@ -3032,6 +3032,82 @@ def improper(self):
                 "improper only makes sense for a group with exactly 4 atoms")
         return topologyobjects.ImproperDihedral(self.ix, self.universe)
 
+    @property
+    def ureybradley(self):
+        """This :class:`AtomGroup` represented as an
+        :class:`MDAnalysis.core.topologyobjects.UreyBradley` object
+
+        Raises
+        ------
+        ValueError
+            If the :class:`AtomGroup` is not length 2
+
+
+        .. versionadded:: 0.21.0
+        """
+        if len(self) != 2:
+            raise ValueError(
+                "urey bradley only makes sense for a group with exactly 2 atoms")
+        return topologyobjects.UreyBradley(self.ix, self.universe)
+
+    @property
+    def cmap(self):
+        """This :class:`AtomGroup` represented as an
+        :class:`MDAnalysis.core.topologyobjects.CMap` object
+
+        Raises
+        ------
+        ValueError
+            If the :class:`AtomGroup` is not length 5
+
+
+        .. versionadded:: 0.21.0
+        """
+        if len(self) != 5:
+            raise ValueError(
+                "cmap only makes sense for a group with exactly 5 atoms")
+        return topologyobjects.CMap(self.ix, self.universe)
+
+
+    def convert_to(self, format):
+        """
+        Convert :class:`AtomGroup` to structure from another library.
+
+        Example
+        -------
+
+        .. code-block:: python
+
+            import MDAnalysis as mda
+            from MDAnalysis.tests.datafiles import GRO
+            u = mda.Universe(GRO)
+            parmed_structure = u.atoms.convert_to('PARMED')
+
+        
+        Parameters
+        ----------
+        format: str
+            The name of the library to convert to, e.g. ``"PARMED"``
+
+
+        Returns
+        -------
+        output:
+            An instance of the structure type from another library.
+        
+        Raises
+        ------
+        TypeError:
+            No converter was found for the required format
+        
+        """
+        try:
+            writer = _CONVERTERS[format]
```

Comment:
Rather than exposing (and pinning down) how we're doing the magic around `_CONVERTERS`, it's better to have a `get_converter_for` method like writers have, where later we can put more/different magic if things change.

Reply:
Thanks, moved.